content/docs/ref/runner.md

```diff
@@ -73,8 +73,193 @@ Any [generic option](/doc/ref) in addition to:
   (you'd need to write your code to save intermediate results to take advantage
   of this).
 
+
+## Cloud Permissions
```

Comment:
kewl... bus seems like it could go under

```
### Using `--cloud-permission-set`
```

without needing a separate section title?